lib/ex_doc/language/erlang.ex

```diff
@@ -728,10 +728,6 @@ defmodule ExDoc.Language.Erlang do
   defp trim_offset(binary, offset) do
     binary
     |> String.trim()
-    |> String.split("\n")
-    |> Enum.map(fn line ->
-      binary_part(line, offset, byte_size(line) - offset)
-    end)
-    |> Enum.join("\n")
+    |> then(&binary_part(&1, offset, byte_size(&1) - offset))
```

Comment:
I think we can remove this function then and call `String.trim_leading(string, "-type ")` instead?

Reply:
Fixed I think